mesheryctl/internal/cli/root/system/stop.go

```diff
@@ -177,11 +178,20 @@ func stop() error {
 			if err = deleteNs(utils.MesheryNamespace, client.KubeClient); err != nil {
 				return err
 			}
+			// Wait for the namespace to be deleted
+			done, err := mesheryNsDeleteCheck()
+			if err != nil {
+				log.Info(err)
+			} else if !done {
+				log.Info("Meshery is taking too long to stop.\nPlease check the status of the pods by executing “mesheryctl system status”.")
```

Comment:
These two conditions can be merged! The error will normally be thrown by `wait` once the timeout is expired.

Reply:
Ah okay. Will do